package.json

```diff
@@ -36,6 +36,7 @@
     "next-auth": "^4.24.5",
     "next-runtime-env": "^3.2.0",
     "react": "^18",
+    "react-datepicker": "^7.2.0",
```

Comment:
suggestion: If it is not too much work, perhaps you could replace react-datepicker to [shadcn date picker](https://ui.shadcn.com/docs/components/date-picker), to stick to a single component lib provider.